src/component/BellDisplay.tsx

```diff
@@ -1,30 +1,15 @@
-import React, { useState } from 'react';
+import React from 'react';
 
 import { css } from 'emotion';
 import MobileCheck from '../util/MobileCheck';
 
 export default function BellDisplay(props: any) {
-  const { amount, imgSrc, alt, bellUnits } = props;
+  const { amount, imgSrc, alt, bellUnits, tooltipName, shownTooltipName, setShownTooltipName } = props;
```

Comment:
Maybe we can add a unit test for this new functionality to check whether shown tooltip can be changed to another tooltip. I'm thinking we could add this test to parent component `TipDisplay`'s test suite since it'll involve interaction between `TipDisplay` and 2 `BellDisplay`s. What do you think?

Reply:
Added a test for this :)